add grid information to readme (#3)
MeteoSwiss/opendata-forecast-data: README.md

### 2.2 Available Parameters

Users can find information about available parameters, including metadata, in the collections listed above.

### 2.3 Accessing Forecast Data

The user can access the forecast model output data from the last 24 hours. Data older than 24 hours is no longer available. To check available data, the user can refer to the relevant collections listed in the table above.

> ### 2.4 Additional Data Information

Reviewer: *Additional Data Information* should be placed at end of the list, after *Grid description*

Users can find information about available parameters, including metadata, in the collection level assets of the above collections.

#### 2.2.1 Parameter metadata

The parameter metadata is part of each GRIB file.

### 2.3 Accessing Forecast Data

The user can access the forecast model data from the last 24 hours. Data older than 24 hours is no longer available. The data in each collection is described in the table above.

> ### 2.4 3D grid structure and representation

Reviewer: First letter should be everywhere uppercase, same for 2.2.1 Parameter Metadata
```suggestion
### 2.4 3D Grid Structure and Representation
```

> The ICON model is an unstructured native grid. It distingiushes between a horizontal and vertical grid structure.

Reviewer: I had a comment and a typo correction about this paragraph see https://github.com/MeteoSwiss/opendata-forecast-data/pull/3#discussion_r1973466224

Combining the two structures results in a 3 dimensional grid over Switzerland and
its surroundings. We differentiate between single and multi level parameters where the
former describes a parameter on a single vertical level and the latter a parameter on
the entire 3-dimensional grid. For example the vertical velocity is stored in multiple vertical
levels, whereas the two-meter-temperature has only information in one vertical level.

#### 2.4.1 Vertical grid

The vertical grid is a height based coordinate system that follows the terrain. It is divided into multiple layers. The closer the layer is to
the surface, the narrower the layers are, as one can see in the picture below.

> Note that the so-called *half levels* correspond to the horizontal grid points, while the *full levels* describe an avarages value

Reviewer: typo
```suggestion
Note that the so-called *half levels* correspond to the horizontal grid points, while the *full levels* describe an averaged value
```

See [jupyter-notebook examples](https://github.com/MeteoSwiss/opendata-nwp-demos).

> ### 2.5 Grid description

Reviewer: ```suggestion
### 2.5 3D Grid Structure and Representation
```

Reviewer: Section 2.4.2 Coordinate System is now redundant with 2.5 Grid Description. I suggest removing it and instead integrating the explanation into the Grid Description section.

Specifically, we should clarify that the ICON native grid is unstructured and we will provide the entire collection of grid points, each with its corresponding latitude and longitude coordinates. This information is available through the CLON (longitude) and CLAT (latitude) variables, which users can retrieve from the dataset assets.

> The ICON model distingiushes between a horizontal and vertical grid structure.

Reviewer: ```suggestion
The ICON model distinguishes between a horizontal and vertical grid structure.
```

> The ICON model distingiushes between a horizontal and vertical grid structure.
> Combining the two structures results in a 3 dimensional grid over Switzerland and
> its surroundings. We differentiate between single and multi level parameters where the
> former describes a parameter on a single vertical level and the latter a parameter on
> the entire 3-dimensional grid. For example the vertical velocity is stored in multiple vertical
> levels, whereas the two-meter-temperature has only information in one vertical level.

Reviewer: ```suggestion
The model data is structured on both a horizontal and vertical grid. While some parameters extend across the entire three-dimensional grid, others are only available at specific vertical levels.
Parameters are classified as either **single-level** or **multi-level**:

- **Single-level parameters** contain data at a specific vertical level.
- **Multi-level parameters** extend across multiple vertical layers.
For example, vertical velocity is stored at multiple vertical levels, while the two-meter temperature is available only at a single vertical level.
```

#### 2.5.1 Vertical grid

The vertical grid is a height based coordinate system that follows the terrain. It is divided into multiple layers. The closer the layer is to

> the surface, the narrower the layers are, as one can see in the picture below.

Reviewer: ```suggestion
the surface, the narrower the layers are, as shown in the image below.
```

> Note that the so-called *halve levels* correspond to the horizontal grid points, while the *full levels* describe an avarges value

Reviewer: All *halve* need to be replaced by *half*
```suggestion
Note that the so-called *half levels* correspond to the horizontal grid points, while the *full levels* describe an avarges value
```

> Note that the so-called *halve levels* correspond to the horizontal grid points, while the *full levels* describe an avarges value

Reviewer: ```suggestion
The so-called half levels align with horizontal grid points, while the full levels represent an averaged value over a vertical interval.
```

> over the whole vertical intervall. In total there exists 80 discrete halve levels.

Reviewer: ```suggestion
over the whole vertical intervall. There are 80 discrete half levels in total.
```

> ![image](Images/VerticalLayers.png)*Illustration of ICON's vertical levels, Working with the ICON Model 2024, Figure 3.2*

Reviewer: Is it possible to reduce the size of the image?

> Be aware that some data is stored on halve and some on full levels.

Reviewer: ```suggestion
Be aware that some data is stored at half levels, hile others are stored at full levels.
```

Reviewer: I think we discussed last Thursday that W is the **only** parameter staggered in the vertical coordinate

> For example the vertical velocity W has its information on halve levels, meaning that the value is exact in this point

Reviewer: ```suggestion
For example, vertical velocity (W) is defined at half levels, meaning its value is exact in this point
```

> and not an avarage value over an interval stored in one point. A parameter with this property is called staggered.

Reviewer: are you sure about the averaging between two levels?

Author: I didn't read it, but Petra told me. Is that enough?

Reviewer: Fine, but in that case, I would rephrase the information to clarify this:

> meaning that the value is exact in this point
and not an avarage value over an interval stored in one point.

Additionally, I would replace 'interval' with 'layer' and explain how the concept of a layer differs from that of levels.

> To check wheather a parameter is staggered, one needs to access the parameter metadata. For more

Reviewer: Need to add the GRIB Key which indicates this

Author: Which one is it?

detailed information on the vertical grid, read section 3.4 in [Working with the ICON Model](https://www.dwd.de/DE/leistungen/nwv_icon_tutorial/pdf_einzelbaende/icon_tutorial2024.pdf?__blob=publicationFile&v=3).

#### 2.5.2 Horizontal grid

> The horizontal grid is based on a triangular icosahedral structure, as one can see in the following representation.

Reviewer: ```suggestion
The horizontal grid is based on a triangular icosahedral structure, as illustrated below.
```

![image](Images/IcosahedralGrid.png)\
*Illustration of the grid construction, Working with the ICON Model, Figure 2.1*

Since the provided data is given

> in the native grid, it is important to note that the information on the grid is not stored on the vertices of the triangles. **When retrieving the

Reviewer: ```suggestion
in the native grid, it is important to note that the information on the grid is **not** stored on the vertices of the triangles. When retrieving the
```

> longitude and latitude be aware that they are corresponding to the center of the circumcircle of each triangle.** For more detailed information on

Reviewer: ```suggestion
longitude and latitude, be aware that they correspond to the **center of the circumcircle of each triangle**. For more detailed information on
```

To check wheather a parameter is staggered, one needs to access the parameter metadata. For more
detailed information on the vertical grid, read section 3.4 in [Working with the ICON Model](https://www.dwd.de/DE/leistungen/nwv_icon_tutorial/pdf_einzelbaende/icon_tutorial2024.pdf?__blob=publicationFile&v=3).

#### 2.5.2 Horizontal grid

The horizontal grid is based on a triangular icosahedral structure, as one can see in the following representation.

![image](Images/IcosahedralGrid.png)\
*Illustration of the grid construction, Working with the ICON Model, Figure 2.1*

Since the provided data is given
in the native grid, it is important to note that the information on the grid is not stored on the vertices of the triangles. **When retrieving the
longitude and latitude be aware that they are corresponding to the center of the circumcircle of each triangle.** For more detailed information on
the horizontal grid, read section 2.1 in [Working with the ICON Model](https://www.dwd.de/DE/leistungen/nwv_icon_tutorial/pdf_einzelbaende/icon_tutorial2024.pdf?__blob=publicationFile&v=3).

### Static GRIB files

#### CLON/CLAT

> #### HHL

Reviewer: Is this final? If it's to be completed, add it in another PR, thanks!
